src/core/js/views/pageView.js

```diff
@@ -17,6 +17,17 @@ define([
             this.disableAnimation = Adapt.config.has('_disableAnimation') ? Adapt.config.get('_disableAnimation') : false;
             this.$el.css('opacity', 0);
             this.listenTo(this.model, 'change:_isReady', this.isReady);
+
+            var accessibility = Adapt.config.get('_accessibility');
+            if (accessibility._isEnabled || accessibility._isEnabledOnTouchDevices) {
+                // create aria-label outside of #wrapper
+                this.pageLabel = document.createElement('div');
+                this.pageLabel.setAttribute('class', 'aria-label relative a11y-ignore-focus prevent-default')  ;
+                this.pageLabel.setAttribute('tabindex', 0);
+                this.pageLabel.setAttribute('role', 'region');
+                this.pageLabel.textContent = Adapt.course.get('_globals')._accessibility._ariaLabels.pageEnd;
+                document.body.appendChild(this.pageLabel);
+            }
```

Comment:
```javascript
$("<div>", {
  class: 'aria-label relative a11y-ignore-focus prevent-default',
  tabindex: 0,
  role: 'region'
})
.html(Adapt.course.get('_globals')._accessibility._ariaLabels.pageEnd)
.appendTo($("body"));
 ```
?

Reply:
Yes, I know, native should be save though?
If you prefere it due to legibility I am fine to update it.

Reply:
Can you update it? It's a 49% character saving (including whitespace)? ;)

Reply:
well that's an argument. Sure will update it.